-- Remove upper bound on uv_build version to avoid breakage when uv_build is updated in ports.

--- pyproject.toml.orig	2026-09-07 15:14:17 UTC
+++ pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["uv_build>=0.12.0,<0.13.0"]
+requires = ["uv_build>=0.12.0"]
 build-backend = "uv_build"
 
 [project]
